resources/user-agents/browsers/opera-mini/opera-mini-10-0-on.json

```diff
@@ -0,0 +1,39 @@
+{
+  "division": "Opera Mini #MAJORVER#.#MINORVER#",
+  "versions": [28, 27, 26, 25, 24, 23, 22, 21, "20.1", 20, 19, 18, 17, 16, 15, 14, 13, 12, 11, "10.2", "10.1", 10],
+  "sortIndex": 1931,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Opera Mini #MAJORVER#.#MINORVER#",
+      "engine": "Presto_2_8",
```

Comment:
According to wikipedia, Opera Mini still uses Presto as of March 2015: https://en.wikipedia.org/wiki/Opera_Mini#Standards_support

However, at the same time it says it uses a mix of Blink and Presto (side bar): https://en.wikipedia.org/wiki/Opera_Mini

So, hard to say, typically Opera Mini can/will be server rendered using Opera's proxy servers.  Sounds like it uses Presto in that case, but maybe uses Blink for on-device rendering?